src/button/interfaces.ts

```diff
@@ -162,6 +119,53 @@ export interface ButtonProps extends BaseComponentProps {
   fullWidth?: boolean;
```

Comment:
Do we have use cases for full width Toggle Buttons?

Reply:
No. Updated